### PR TITLE
fix links to samples

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/persistence.md [hunk 1]
```diff
@@ -646,18 +646,18 @@ reference documentation of the chosen plugin.
 
 ## Example project
 
-@java[@extref[Persistence example project](samples:akka-sample-persistence-java)]
-@scala[@extref[Persistence example project](samples:akka-sample-persistence-scala)]
+@java[@extref[Persistence example project](samples:akka-samples-persistence-java)]
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/persistence.md]: Wait, so in https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf it is `akka-sample-persistence-java` and the directory in https://github.com/akka/akka-samples/tree/2.6/akka-sample-persistence-java is `akka-sample-persistence-java` - but indeed https://developer.lightbend.com/start/?group=akka&project=akka-samples-persistence-java actually works

where does `akka-samples-persistence-java` come from?

Comment (patriknw) [reply akka-docs/src/main/paradox/typed/persistence.md]: I don't know how it works. Maybe example-code-service is concatenating the names? 

https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf#L3

https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf#L98

**Comment by jlprat** [comment]
There is another pair of occurrences in [actors.md](https://github.com/akka/akka/blob/master/akka-docs/src/main/paradox/typed/actors.md):
Lines: 1065 and 1066
This
```
Hakkers](http://www.lightbend.com/activator/template/akka-sample-fsm-scala)).] @java[[Dining
Hakkers](http://www.lightbend.com/activator/template/akka-sample-fsm-java-lambda)).] It will replace the current behavior (i.e. the top of the behavior
```
should be:

```
Hakkers](http://www.lightbend.com/activator/template/akka-samples-fsm-scala)).] @java[[Dining
Hakkers](http://www.lightbend.com/activator/template/akka-samples-fsm-java-lambda)).] It will replace the current behavior (i.e. the top of the behavior
```

**Comment by patriknw** [comment]
wow, activator. thanks @jlprat

**Comment by jlprat** [comment]
No problem, I saw the failing links and I was trying to fix it, and then I saw your PR :laughing:

**Comment by patriknw** [comment]
fixed, by removing: 0eb5f3e783

**Review by jlprat — APPROVED** [review]
LGTM :+1:

**Comment by patriknw** [comment]
Do we think this is important enough to patch the html for 2.6.0? maybe the examples.html page since we linked to that from announcement?

**Comment by jlprat** [comment]
My humble opinion: if it is expected to attract new people who are not familiar with the ecosystem (Lightbend and templates), yes it is important enough.

**Comment by patriknw** [comment]
I have patched the html of the pages in the first commit here. For example https://doc.akka.io/docs/akka/current/project/examples.html

**Comment by jlprat** [comment]
I can confirm it works :+1:

**Pull request merged**